src/db/migrations/20200304100142-create-comment.js

```diff
@@ -13,8 +13,8 @@ module.exports = {
       references: {
         model: 'Requests',
         key: 'id',
-        as: 'id',
-      },
+        as: 'id'
```

Comment:
why did you edit this file?

Reply:
Same as the other ones, Once I open the file there are requesting to fix eslint lint